more/free-programming-interactive-tutorials-en.md

```diff
@@ -219,17 +220,22 @@
 
 #### jQuery
 
-* [Front End Libraries: jQuery](https://www.freecodecamp.org/learn/front-end-libraries/jquery)
+* [Front End Development Libraries Certification: jQuery](https://www.freecodecamp.org/learn/front-end-libraries/#jquery) - freeCodeCamp
 * [jQuery Tutorial](https://www.w3schools.com/jquery/) - W3Schools
 
 
 #### React
 
-* [Front End Libraries: React](https://www.freecodecamp.org/learn/front-end-libraries/react)
+* [Front End Development Libraries Certification: React](https://www.freecodecamp.org/learn/front-end-libraries/#react) - freeCodeCamp
 * [React Tutorial](https://react-tutorial.app)
 * [React Tutorial](https://www.w3schools.com/react/) - W3Schools
 
 
+#### Redux
+
+* [Front End Development Libraries: Redux](https://www.freecodecamp.org/learn/front-end-development-libraries/#redux) - freeCodeCamp
```

Comment:
Use each course url instead of its track from belongs to

```suggestion
* [Front End Development Libraries: Redux](https://www.freecodecamp.org/learn/front-end-development-libraries/redux) - freeCodeCamp
```